specs/fjord/derivation.md

```diff
@@ -48,3 +54,49 @@ be in the same epoch as the the last pre-Fjord blocks, even if these blocks woul
 have these L1-origin timestamps according to pre-Fjord rules. So the same L1 timestamp would be
 shared within a pre- and post-Fjord mixed epoch. This is considered a feature and is not considered
 a security issue.
+
+## Increasing `MAX_RLP_BYTES_PER_CHANNEL` and `MAX_CHANNEL_BANK_SIZE`
+
+With Fjord, `MAX_RLP_BYTES_PER_CHANNEL` will be increased from 10,000,000 bytes to 100,000,000 bytes,
+and `MAX_CHANNEL_BANK_SIZE` will be increased from 100,000,000 bytes to 1,000,000,000 bytes.
+
+The usage of `MAX_RLP_BYTES_PER_CHANNEL` is defined in [Channel Format](../protocol/derivation.md#channel-format).
+The usage of `MAX_CHANNEL_BANK_SIZE` is defined in [Channel Bank Pruning](../protocol/derivation.md#pruning).
+
+The new value will be used when the timestamp of the L1 origin of the derivation pipeline >= the Fjord activation
+timestamp. (TODO: Link to L1 origin activation like in Canyon CB changes).
```

Comment:
Should I maybe specify this too in my seq drift constants specs PR?

Reply:
Yea, spec'ing the activation should probably be done, but we can share it. I'm planning on re-using the wording from Canyon which has already done this

Reply:
Sounds good, let's share. The sequencer drift spec PR got merged.